src/frontend/src/icp-eth/services/eth.services.ts

```diff
@@ -89,7 +90,7 @@ const loadCkErc20PendingTransactions = async ({
 } & IcCkLinkedAssets) => {
 	const logsTopics = (to: EthAddress): (string | null)[] => [
 		CKERC20_HELPER_CONTRACT_SIGNATURE,
-		null,
+		(twinToken as Erc20Token).address,
```

Comment:
Cool, that easy? Nice 👍

NTH: `(twinToken as Erc20Token)?.address ?? null` just in case **** happens some day?

Reply:
correct! theoretically all the tokens that arrive there are of type `Erc20Token`, but...